<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1431</width>
    <height>1109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="editPosY"/>
        </item>
        <item>
         <widget class="QLineEdit" name="editPosX"/>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0" colspan="2">
                <widget class="QLabel" name="label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>点击按钮获取坐标，按</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2" colspan="2">
                <widget class="QLineEdit" name="stop">
                 <property name="text">
                  <string>CTRL+R</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QLabel" name="label_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>停止</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="altButton">
                 <property name="text">
                  <string>改造石</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1" colspan="2">
                <widget class="QLineEdit" name="alt_x">
                 <property name="text">
                  <string>808</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="3" colspan="2">
                <widget class="QLineEdit" name="alt_y">
                 <property name="text">
                  <string>671</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QPushButton" name="augButton">
                 <property name="text">
                  <string>增幅石</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1" colspan="2">
                <widget class="QLineEdit" name="aug_x">
                 <property name="text">
                  <string>994</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="3" colspan="2">
                <widget class="QLineEdit" name="aug_y">
                 <property name="text">
                  <string>765</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QPushButton" name="equButton">
                 <property name="text">
                  <string>装备</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1" colspan="2">
                <widget class="QLineEdit" name="equ_x">
                 <property name="text">
                  <string>1121</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="3" colspan="2">
                <widget class="QLineEdit" name="equ_y">
                 <property name="text">
                  <string>934</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QPushButton" name="pushButton_3">
                 <property name="text">
                  <string>富豪石</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1" colspan="2">
                <widget class="QLineEdit" name="lineEdit_8"/>
               </item>
               <item row="4" column="3" colspan="2">
                <widget class="QLineEdit" name="lineEdit_10"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>改造石剩余</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="altNum">
                 <property name="text">
                  <string>400</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLineEdit" name="altUsed">
                 <property name="text">
                  <string>改造石使用：</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>增幅石剩余</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="augNum">
                 <property name="text">
                  <string>400</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLineEdit" name="augUsed">
                 <property name="text">
                  <string>增幅石使用：</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>尝试次数</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="attemptedTimes"/>
               </item>
               <item row="2" column="2" rowspan="2">
                <widget class="QPushButton" name="startButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>开始</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>目标词缀包含</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="target">
                 <property name="text">
                  <string>+1</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="text">
           <string>F10</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonMouseLeftClick">
          <property name="text">
           <string>鼠标左键</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonMouseRightClick">
          <property name="text">
           <string>鼠标右键</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonGetPos">
          <property name="text">
           <string>获取坐标</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="text">
           <string>F7</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="MyLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>440</y>
          <width>231</width>
          <height>20</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string notr="true">按下快捷键</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>440</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
